config/platforms/css.ts

```diff
@@ -2,21 +2,41 @@ import StyleDictionary from 'style-dictionary'
 import {PlatformInitializer} from '~/types/PlatformInitializer'
 import {isSource} from '~/config/filters'
 
-export const css: PlatformInitializer = (outputFile, prefix, buildPath, _options): StyleDictionary.Platform => ({
-  prefix,
-  buildPath,
-  transforms: ['name/cti/kebab', 'color/hex', 'color/hexAlpha', 'shadow/css'],
-  options: {
-    basePxFontSize: 16
-  },
-  files: [
-    {
-      destination: `${outputFile}`,
-      format: `css/variables`,
-      filter: isSource,
-      options: {
-        outputReferences: false
+const getCssSelectors = (outputFile: string): {selector: string; selectorLight: string; selectorDark: string} => {
+  // check for dark in the beginning of the output filename
+  const lastSlash = outputFile.lastIndexOf('/')
+  const outputBasename = outputFile.substring(lastSlash + 1, outputFile.indexOf('.'))
+  const themeName = outputBasename.replace('-', '_')
```

Comment:
Noticed one small bug testing this with @jonrohan– this only seems to replace the first `-`, but the whole theme name should use underscores like `dark_high_contrast`. We tried to fix this, but ran into typescript issues and couldn't solve it 😅

Reply:
Ahh, good catch, will fix it.